resources/src/main/java/org/robolectric/res/android/CppAssetManager2.java

```diff
@@ -850,8 +847,7 @@ public ApkAssetsCookie GetResource(int resid, boolean may_be_bag,
       final Ref<ResTable_config> out_selected_config,
       final Ref<Integer> out_flags) {
     final Ref<FindEntryResult> entry = new Ref<>(null);
-    ApkAssetsCookie cookie =
-        FindEntry(resid, density_override, false /* stop_at_first_match */, entry);
+    ApkAssetsCookie cookie = FindEntry(resid, density_override, /* stop_at_first_match */ entry);
```

Comment:
Remove this `stop_at_first_match` comment.

Reply:
interesting, I think I've already removed it and then pushed the code. Sorry for the same mistake. I'll check tonight